definitions/types/aws-security-iam.json

```
@@ -5,7 +5,7 @@
   "description": "IAM Policies",
   "additionalProperties": false,
   "patternProperties": {
-    "^[a-z\/]+$": {
+    "^[a-z\/-]+$": {
```

```suggestion
    "^[a-z-\/]+$": {
```
Gonna suggest moving it to the front of the character class so if we add any more to the end, the `-` doesn't get interpreted as a range. I've had that strange issue before where it was `[...$-&]` and it reads it as "all symbols between `$` and `&`". Very subtle.

---

Very good suggestion here! Thank you